package.json

```diff
@@ -5,7 +5,10 @@
   "author": "Fluid Project",
   "bugs": "https://issues.fluidproject.org/browse/FLOE/component/11615",
   "homepage": "http://www.fluidproject.org/",
-  "dependencies": {},
+  "dependencies": {
+      "d3":"latest",
```

Comment:
In the case of Flocking, you'll have to pin to a particular commit hash until v0.2.0 is released. I recommend colinbdclark/flocking#543b339f8aac1c10bffc5e70344f56bc63cc25b1.